setup.py

```diff
@@ -13,7 +13,7 @@
 
 requirements = [
     "numpy>=1.16.3",
-    "scipy>=1.2.0",
+    "scipy>=1.2.0,<1.14.0",
```

Comment:
What breaks in SciPy `1.14`?  The latest release is `1.16.2`, so we should work on that.  (this is outside the scope of this PR)

Reply:
It looks like @AaronDJohnson advised that you can remove the maximum version in [this comment](https://github.com/nanograv/enterprise_extensions/pull/242#issuecomment-2527143032)